fe1-web/model/objects/WalletCryptographyHandler.ts

```diff
@@ -1,12 +1,27 @@
-import { IndexedDBStore, WalletCryptoKey } from 'store/stores/IndexedDBStore';
+import { get, set, update } from 'idb-keyval';
 import STRINGS from 'res/strings';
 
+/* wallet cryptography key interface */
+export interface WalletCryptoKey {
+  privateKey: CryptoKey,
+  publicKey: CryptoKey,
+}
+
 /**
  * This class has the job of handling the cryptography functions of the wallet.
  * It interacts with the IndexedDB database in order to store and retrieve the
- * secret key. It also encrypts and decrypts the tokens with the retrieved key.
+ * secret key. It also encrypts and decrypts the wallet seed (used to create the
+ * tokens) with the retrieved key.
+ * More info on this approach at https://blog.engelke.com/2014/09/19/saving-cryptographic-keys-in-the-browser/
  */
 export class WalletCryptographyHandler {
+  /* the crypto library - passed to constructor */
+  private cryptography!: object;
```

Comment:
you shouldn't need to assert that it's not-null with `!` here, right?
`object` is by definition not-null :)

Reply:
you might also want to reference the actual interface: `Crypto`
https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_dom_d_.crypto.html#subtle

which itself contains `SubtleCrypto` 
https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_dom_d_.subtlecrypto.html